src/compo/modis_aod2ioda.py

```diff
@@ -133,14 +156,20 @@ def _read(self):
             over_land = np.logical_not(land_sea_flag == 0)
             UNC = np.where(over_land, unc_land, np.add(0.05, np.multiply(0.15, aod)))
```

Comment:
Do we understand what this is?

Reply:
The product only provides estimated uncertainty over land, so it uses `0.05 + 0.15 * AOD`  for ocean points.
But I don't know if there is any reference about it.

Reply:
@jeromebarre The formula for uncertainty is from Levy et al. 2010 https://acp.copernicus.org/articles/10/10399/2010/acp-10-10399-2010.pdf (page 4, right column)
and it cited Chu et al. 2002 https://agupubs.onlinelibrary.wiley.com/doi/epdf/10.1029/2001GL013205
However, the results are from the comparison with AERONET over land.
It could be overestimated over the ocean if we think the error over the ocean should be smaller than over land.